### DISCUSSION TITLE
Add Bengali (bn-BD) translation

### DISCUSSION THREAD

**Discussion by ghost** [chipzoller/hugo-clarity#143] [General]
'Bengali' or 'Bangla' is the 7th most spoken language in the world according to [WorldAtlas](https://www.worldatlas.com/articles/most-popular-languages-in-the-world.html). I think Bengali would be a cool addition to the available translations in **Hugo Clarity**. As a Bengali speaking person, it would be fairly easy for me to add Bengali translation. I would like to submit a `Pull Request` when I'm finished translating if you (The Maintainers) don't mind.

**Comment by onweru** [comment]
@ghost 

> I would like to submit a Pull Request when I'm finished translating if you (The Maintainers) don't mind.

We don't mind. Please proceed.
